Alignment/OfflineValidation/plugins/TrackerGeometryCompare.cc

```diff
@@ -569,7 +587,15 @@ void TrackerGeometryCompare::compareSurfaceDeformations(TTree* refTree, TTree* c
           if (TMath::Abs(surfDeform_[npar]) > (m_rangeHigh_ - m_rangeLow_) / (10. * m_nBins_))
             m_h1_[histname2.str()]->Fill(surfDeform_[npar]);
         }
-        (m_vtkmap_.at(npar)).fill_current_val(inputRawid2, surfDeform_[npar]);
+        if (phase_ < SiPixelPI::phase::two) {
+          if (phase_ == SiPixelPI::phase::zero) {
+            (m_vtkmap_.at(npar)).fill_current_val(inputRawid2, surfDeform_[npar]);
+          } else {
+            if (inputSubdetid2 != 1 && inputSubdetid2 != 2) {
+              (m_vtkmap_.at(npar)).fill_current_val(inputRawid2, surfDeform_[npar]);
```

Comment:
leave a comment that the "old-style" Tracker Map does not support Phase-1 pixels ?